BraveRewardsUI/Supporting Files/Strings.swift

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+struct Strings {
+  
+  struct DateTime {
```

Comment:
I was thinking of splitting some strings in nested structs instead of just putting everything in one place, thoughts?